### ISSUE TITLE
NW.js crashes on macOS High Sierra (10.13.6)

### ISSUE THREAD

**Issue by sysrage** [nwjs/nw.js#7946]
### Issue Type

- [x] Bug Report
- [x] Successfully reproduced against the latest version of NW.js?

### Current/Missing Behavior

NW.js crashes when run in macOS High Sierra (10.13.6). This happens whether run by itself (expecting the default window to appear) or run with an appropriate `package.json`.

When running an NW.js app through Finder, it looks like nwjs is running but no windows appear. When running `nwjs.app/Content/MacOS/nwjs` from the terminal, it reports the following:

```
../nwjs-sdk-v0.64.1-osx-x64/nwjs.app/Contents/MacOS/nwjs .
[0817/201011.282098:ERROR:zip.cc(179)] Cannot open '/Users/sysrage/Downloads/nwjs-sdk-v0.64.1-osx-x64/nwjs.app/Contents/Resources/app.nw': FILE_ERROR_NOT_FOUND: No such file or directory (2)
[0817/201011.282801:ERROR:zip.cc(179)] Cannot open '/Users/sysrage/Downloads/nwjs-sdk-v0.64.1-osx-x64/package.nw': FILE_ERROR_NOT_FOUND: No such file or directory (2)
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/sysrage/Downloads/nwjs-sdk-v0.64.1-osx-x64/nwjs.app/Contents/Frameworks/nwjs Framework.framework/Versions/101.0.4951.67/libnode.dylib **(which was built for Mac OS X 10.15)**
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/sysrage/Downloads/nwjs-sdk-v0.64.1-osx-x64/nwjs.app/Contents/Frameworks/nwjs Framework.framework/Versions/101.0.4951.67/libnode.dylib **(which was built for Mac OS X 10.15)**
  Expected in: /usr/lib/libSystem.B.dylib

Received signal 6
 [0x000110516099]
 [0x0001104748e3]
 [0x000110515ff1]
 [0x7fff7d42cf5a]
 [0x00010c7d250b]
 [0x00010c7dcbe9]
 [0x00010c7b0482]
 [0x00010c7b05a9]
 [0x7fff7d11e292]
 [0x00010a95fb80]
 [0x000109205c71]
 [0x0001092d1487]
 [0x00011568342c]
 [0x000115683ad9]
 [0x000110002d6f]
 [0x000110003707]
 [0x000110001d2f]
 [0x00011000265c]
 [0x00010c84d2fe]
 [0x00010577da1c]
 [0x7fff7d11e015]
[end of stack trace]
[0817/201012.062013:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffeea482000, 0x2000): (os/kern) invalid address (1)
```

### Expected/Proposed Behavior

NW.js should work on High Sierra, unless I missed the memo of non-support. It would also be helpful to list the minimum required macOS version on the NW.js download page.

### Additional Info

- Operating System: macOS High Sierra (10.13.6)
- NW.js Version: 0.64.1 through 0.67.1 all fail

**Comment by Blatman** [comment]
Similar to #7874 - it appears that node 18 requires 10.15 or later so unless NWjs can be built with less than node 18 then only Catalina and above is supported. It seems to have slipped under the radar but I reckon the min version key of the plist should now be set to 10.15 - annoying for me since my main Dev machine is 10.13.
Can't quite remember but strangely enough >= 0.67.1 may have worked on Mojave so it may be a case that below 10.15 that is supported by node it may or may not work.

**Comment by sysrage** [comment]
@rogerwang per your comment in the issue Blatman linked, this is apparently expected. I recommend adding a note to the downloads page and/or elsewhere, indicating the minimum required OS version.

**Comment by sysrage** [comment]
Further detail on the plist issue:

`nwjs-sdk-v0.67.1-osx-x64\nwjs.app\Contents\Info.plist`:
```xml
	<key>LSMinimumSystemVersion</key>
	<string>10.13</string>
```
This should be 10.15, so the OS properly prevents users from attempting to run it. Right now, they can run it but it "crashes". If that field is set correctly, the OS won't let them run it at all.

**Comment by gmarinov** [comment]
node 18 itself works fine on macos mojave 10.14. 

here's a screenshot from a just-downloaded 0.72 on macos 10.14.6

<img width="748" alt="Screenshot 2023-01-22 at 19 57 04" src="https://user-images.githubusercontent.com/14101296/213937448-703ae40d-92da-4055-a24f-278be70d0f45.png">

**Comment by rozek** [comment]
Just in case that anybody reads this:

NW.js 0.64.0 is the lastest version I got running under macOS 10.13.6 (High Sierra) on my old Mac mini mid-2011

**Comment by Blatman** [comment]
Yup for sure 10.13.6 will not run 0.64.1 or above as the prebuilt binaries use node 18 which is minimum 10.15 support.
Maybe there is a case for the plist min-version to be set for 10.14 since Mojave seems to largely work OK and there is as yet no identified fatal crashes as there is with 10.13 but it does look like node is going to follow the Apple supported releases going forward.
